dataproc_jupyter_plugin/utils/utilities.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Google LLC
```

Comment:
Please delete this file. This is both a terrible name for a package, and it reimplements functionality we already have from the `google.cloud.jupyter_config` package.

Reply:
Deleted